### PR TITLE
[AutoGen Studio] Allow multiline strings in chat input

### PR DESCRIPTION
**Pull request by dragosMC91**
## Why are these changes needed?

It is useful for users to be able to input multiline strings when passing prompts to a LLM. This is mostly beneficial for the user, but it could also enhance the prompt since clear delimiters might yield better responses from the AI model: https://platform.openai.com/docs/guides/prompt-engineering/tactic-use-delimiters-to-clearly-indicate-distinct-parts-of-the-input.
This is currently rather hard to do since the entire input string is crammed up on a single line, and reviewing + editing it before submission is difficult.

## Changes introduced in this PR:
1. the `input` was replaced with `textarea`
2. `Shift+Enter` key combination no longer submits the input, rather it creates a new line so the user can continue typing before pressing `Enter` to submit the prompt
3. the `textarea` component can be expanded to a maximum of 120px so the user can more easily tweak his prompt before submission

**Note:** most chat interfaces behave like this, I just wanted the same behavior here (see https://chat.openai.com/ for example)

## Demo

https://github.com/microsoft/autogen/assets/11348201/87c883d6-1faa-4348-85c9-a562c998d38a

## Checks

- [x] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by dragosMC91** [comment]
@microsoft-github-policy-service agree

**Review by victordibia — APPROVED** [review]

File: samples/apps/autogen-studio/frontend/src/components/header.tsx
Comment (victordibia) [inline samples/apps/autogen-studio/frontend/src/components/header.tsx:1]: minor (unrelated) fix for visual consistency in the headers

File: samples/apps/autogen-studio/frontend/src/components/views/playground/chatbox.tsx
[the remark this answers is not in the document]
Comment (dragosMC91) [reply samples/apps/autogen-studio/frontend/src/components/views/playground/chatbox.tsx:1]: thank you very much @victordibia !
btw, I noticed while editing this component that there is not linter, some lines had multiple consecutive whitespaces, or indentation was not correct on others.
Is there any plan to add a linter in the future ? Should I add one in a different PR?

[the remark this answers is not in the document]
Comment (rickyloynd-microsoft) [reply samples/apps/autogen-studio/frontend/src/components/views/playground/chatbox.tsx:1]: Our PRs require pre-checkin, which runs the black linter.

**Pull request merged**